src/interface.jl

```diff
@@ -73,7 +75,7 @@ end
 
 
 function AFM(R::Real)
-   Interface(:d, :f, (ϵ,σ)->(d = (R*(3. /4.).^(2. /3)) .* ϵ.^(2. /3), f = R.^2 .* σ ), (d,f)->(ϵ = 4. /(3. *R^1.5) .* d.^1.5, σ = f./R^2) )
+   Interface(:d, :f, (ϵ,σ)->(d = (R*(3. /4.)^(2. /3)) .* ϵ.^(2. /3), f = R^2 .* σ ), (d,f)->(ϵ = 4. /(3. *R^1.5) .* d.^1.5, σ = f./R^2) )
```

Comment:
Any reason why we `3. / 4. ` is better than just `3/4`? Also, why sometimes `x. / y.` and other times just `x. / y`?

Reply:
I guess I'm a bit paranoid with making sure is it not interpreted as an integer division... It doesn't seem to be a problem in Julia though.